components/org.wso2.carbon.identity.api.server.action.management/org.wso2.carbon.identity.api.server.action.management.v1/src/main/java/org/wso2/carbon/identity/api/server/action/management/v1/util/ActionMgtEndpointUtil.java

```diff
@@ -91,6 +93,28 @@ public static APIError handleActionMgtException(ActionMgtException e) {
         return handleException(status, errorCode, e.getMessage(), e.getDescription());
     }
 
+    public static ActionMgtServerException buildActionMgtServerException(ActionMgtEndpointConstants.ErrorMessage error,
```

Comment:
Lets add doc comments.

Reply:
I don't think it's necessary for this util class which is used just internally. And none of the methods have doc comments. If we are adding we have to improve all